### PR TITLE
updated things to mostly work when a config file is not writeable

### PR DESCRIPTION
**Pull request by lwahlmeier**
This PR fixes issue #27

### CODE REVIEW

File: stim/rootcmd.go [hunk 1]
```diff
@@ -28,6 +29,8 @@ func initRootCommand(viper *viper.Viper) *cobra.Command {
 	viper.BindPFlag("noprompt", cmd.PersistentFlags().Lookup("noprompt"))
 	cmd.PersistentFlags().StringP("auth-method", "", "", "Default authentication method (ex: ldap, github, etc.)")
 	viper.BindPFlag("auth.method", cmd.PersistentFlags().Lookup("auth-method"))
+	cmd.PersistentFlags().StringP("isautomated", "", "false", "Error on anything that needs to prompt and was not passed in as an ENV var or command flag")
```
Comment (bartlettc22) [inline stim/rootcmd.go]: nit-pick but can we do `is-automated` to be consistent with the other vars?

**Comment by bartlettc22** [comment]
Looks good - made 1 comment...

**Review by bartlettc22 — APPROVED** [review]

File: stim/rootcmd.go (hunk 1 above)
Comment (dwilson6) [reply stim/rootcmd.go]: +1

**Review by dwilson6 — APPROVED** [review]

File: stim/rootcmd.go (hunk 1 above)
Comment (lwahlmeier) [reply stim/rootcmd.go]: I originally had it that way but I was worried about how that translates into the config.yaml file.  The only other example of this is auth-method, which ends up as auth.method as the lookup which looks like:
```yaml
auth:
  method: "test"
```
in yaml.  I didnt like is-automated, translating to is.automated, which ends up in the config.yaml as:
```yaml
is:
  automated: true
```
so just moved to just is-automated.  I can have the --is-automated then just not translate the lookup in to the config the same way.  let me know what you guys think.

Comment (bartlettc22) [reply stim/rootcmd.go]: That makes sense.  However, I don't think that dashes in parameters need to always directly translate into child elements in the config.  I could see us having some complex nesting in the config but want to simplify it for the command line parameters. With that said, I'm ok either way as we still need to clean up the config/arguments so we can address it then.

**Pull request merged**